app/connectors/AuthProviderClient.scala

```diff
@@ -47,7 +49,9 @@ trait AuthProviderClient {
 
   val allRoles = List(FaststreamTeamRole, ServiceSupportRole, ServiceAdminRole, SuperAdminRole, TechnicalAdminRole)
 
-  def getRole(roleName: String) = allRoles.find(_.name == roleName).getOrElse(throw new UserRoleDoesNotExist(s"No such role: $roleName"))
+  def getRole(roleName: String) = allRoles.find(_.name == roleName).getOrElse(
```

Comment:
So far, in every case this throw is desireable, if we wanted to have different classes using this method dealing with it differently i'd certainly consider using Try - otherwise it's just over engineering IMHO